app/models/form_profiles/va_0873.rb

```diff
@@ -64,24 +63,40 @@ def initialize_ava_profile
 
     VA0873::FormAvaProfile.new(payload)
   rescue => e
-    log_exception_to_sentry(e, {}, prefill: :personal_information)
-    {}
+    handle_exception(e, :ava_profile)
```

Comment:
Is there any risk of PII being exposed in this Sentry log?

Reply:
None, it'll just log the error. No PII will be in the error message.